### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by myaserkhan**
Following up on my portfolio mobile webpage. I have optimized the previous code with that I have also added two sections.
The first section is about six projects using CSS Grid and the second section is about my skills.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,6 +40,276 @@ <h1 class="primary">Hey there. I'm Yasir <br></h1>
             </div>
         </div>
     </section>
-
+    <section class="works" id="portfolio">
+        <div class="projects-wrapper">
+          <div class="works-header">
+            <h1>My Recent Projects</h1>
```
Comment (phelian23) [inline index.html]: - [x] Kindly make use of a `hx` (h2, h3, ..., h6) for the header of the work section. conventionally, a project should only have one `<h1>` tag for the main title of the page in the case of this project your name. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

File: style.css [hunk 2]
```diff
@@ -109,17 +104,285 @@
...
+  background: #36b37f;
+  border-radius: 4px;
+  border: none;
+  outline: none;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  cursor: pointer;
+  text-transform: capitalize;
+}
```
Comment (phelian23) [inline style.css:228]: - [x] Kindly implement the button interactions (hover and pressed) styling for your buttons

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @MYaserKhan,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Great job setting up the correct linters ✔️
Professional documentation ✔️
Correct workflow ✔️
Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @MYaserKhan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**